src/seedu/addressbook/AddressBook.java

```diff
@@ -1147,11 +1147,11 @@ private static String getUsageInfoForExitCommand() {
      * Removes sign(p/, d/, etc) from parameter string
      *
      * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
-     * @return  string without the sign
+     * @param trim  Parameter trim to be removed
+     * @return  string without the trim
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefixTrim(String s, String trim) {
+        return s.replace(trim, "");
```

Comment:
notice that the method still replaces all occurrences of the prefix pattern. please read the description carefully again